parbase/FairParRootFileIo.cxx

```diff
@@ -84,7 +84,7 @@ void FairParRootFile::readVersions(FairRtdbRun* currentRun)
     // in the ROOT file
     delete run;
 
-    run = static_cast<FairRtdbRun*>(RootFile->Get((const_cast<char*>(currentRun->GetName()))));
+    run = RootFile->Get<FairRtdbRun>(currentRun->GetName());
```

Comment:
There is no check if the run exist. Should be added.

Reply:
dupe, marking resolved.